src/clarity-angular/dropdown/dropdown-menu.ts

```diff
@@ -4,6 +4,11 @@ import {Component} from "@angular/core";
     selector: "clr-dropdown-menu",
     template: `
         <ng-content></ng-content>
-    `
+    `,
+    host: {
+        "[class.dropdown-menu]" : "true",
```

Comment:
Will leave this as is for now for backward compatibility reasons. We should tackle this when we address css naming convention across the board.